internal/test/testdata_test.go

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package test
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/pkg/errors"
+	"go.uber.org/goleak"
+)
+
+func TestGetTestdataPath(t *testing.T) {
+	type args struct {
+		filename string
+	}
+	type want struct {
+		want string
+	}
+	type test struct {
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, string) error
+		beforeFunc func(args)
+		afterFunc  func(args)
+	}
+	defaultCheckFunc := func(w want, got string) error {
+		if !reflect.DeepEqual(got, w.want) {
+			return errors.Errorf("got = %v, want %v", got, w.want)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "get exists test file success",
```

Comment:
Do we need to add test case for `not exist`?

Reply:
even though if we get the file which is not exists, it still return the result. 
do you think we need this test case? (as we process the file path in string not using the filepath package~

Reply:
```suggestion
			name: "returns the result of combining the given path and test directory",
```
sorry. I understand your opinion.
I Imagined, from the test case name, that there is a pattern that doesn't exist.
What about test case names like this?